src/javascript/binary/livechart.js

```diff
@@ -281,7 +281,7 @@ LiveChartTick.prototype.process_data = function(point) {
             epoch: parseInt(point[1]),
             quote: parseFloat(point[2])
         };
-        if (!this.chart && !this.chart.series) {
+        if (!this.chart || !this.chart.series) {
```

Comment:
what difference it will make, i don't its better than before.. Say this.chart is defined and this.chart.series is not defined  then it will not return and this.chart.series[0] will give error then. And we use these conditions at lot of places we need to update them also